rules/scala_proto/private/core.bzl

```diff
@@ -55,19 +55,28 @@ def scala_proto_library_implementation(ctx):
                 "supports-workers": supports_workers,
                 "supports-multiplex-sandboxing": supports_workers,
                 "supports-worker-cancellation": supports_workers,
+                "supports-path-mapping": supports_workers,
             },
         ),
         arguments = [args],
     )
 
+    shell_args = ctx.actions.args()
+    shell_args.add(ctx.executable._zipper)
+    shell_args.add_all([gendir], expand_directories = False)
+    shell_args.add(gendir.short_path)
```

Comment:
Shouldn't this be `shell_args.add([gendir], map_each = _short_path)`?

Reply:
As far as I understand things, `short_path` doesn't include any configuration specific parts of the path, so it doesn't need to be done the same way.

From the state of path mapping thread:
> Generally speaking, all functions that return a path string that may contain a configuration prefix such as bazel-out/darwin-amd64-fastbuild/bin must only be called in map_each callbacks, where they are automatically path mapped by Bazel.

and https://bazel.build/rules/lib/builtins/File#short_path
> The path of this file relative to its root. This excludes the aforementioned root, i.e. configuration-specific fragments of the path. This is also the path under which the file is mapped if it's in the runfiles of a binary.

Reply:
This makes sense. We discussed this over our call and I understand better what `short_path` does now.